src/registry.js

```diff
@@ -132,6 +132,22 @@ exports.traceIs = function(traceType, category) {
     return !!_module.categories[category];
 };
 
+/**
+ * Retrieve component module method
+ *
+ * @param {string} name
+ *  name of component (as declared in component module)
+ * @param {string} method
+ *  name of component module method
+ * @return {function}
+ */
+exports.getComponentMethod = function(name, method) {
```

Comment:
Because, what if the component isn't registered? Then, we would have to add logic like:

```js
var Annotations = Registry.getComponent('annotations');
if(Annotations) Annotations.draw();
```